src/test/resources/results/scriptCommand/testScriptCommandForSchemaEvolution_Alter_Table_Modify_column/Employee/snowpipe-redefinition.sql

```diff
@@ -0,0 +1,60 @@
+USE DATABASE SANDBOX_POC1;
```

Comment:
Since this is case of modify columns and we are ignoring these changes for now I think we should neither generate script folder nor any sql scripts for these tables.